src/layouts/PageLayout/PageLayout.module.scss

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../styling/defs.scss' as bk;
+@use '../../styling/features/typography.scss' as typography;
+
+@layer baklava.components {
+  .bk-page-layout {
+    @include bk.component-base(bk-page-layout);
+
+    display: flex;
+    flex-direction: column;
+    gap: bk.$spacing-7;
+
+    .bk-page-layout__nav {
+      margin-block-start: calc(-1 * (bk.$spacing-7 - bk.$spacing-4));
+
+      ul {
+        margin-block-end: calc(-1 * (bk.$spacing-7 - bk.$spacing-2));
+      }
+    }
+
+    .bk-page-layout__header {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    .bk-page-layout__header__actions {
+      display: flex;
+      flex-direction: row;
+      gap: bk.$spacing-7;
+    }
+
+    .bk-page-layout__header__h3 {
```

Comment:
Nest the two selectors above inside of `bk-page-layout__header`.

Also, rename `h3` here to "heading" so it's not hardcoding the level.